compiler/rustc_const_eval/src/interpret/visitor.rs

```diff
@@ -1,16 +1,13 @@
 //! Visitor for a run-time value with a given layout: Traverse enums, structs and other compound
 //! types until we arrive at the leaves, with custom handling for primitive types.
 
+use super::{InterpCx, MPlaceTy, Machine, Projectable};
```

Comment:
Please don't unnecessarily reorder imports, and in particular don't merge out-of-crate and in-crate imports into a single import "block".